Source/Core/DolphinWX/MemcardManager.cpp

```diff
@@ -430,7 +430,7 @@ bool CMemcardManager::CopyDeleteSwitch(u32 error, int slot)
 		if (slot != -1)
 		{
 			memoryCard[slot]->FixChecksums();
-			if (!memoryCard[slot]->Save()) PanicAlertT(E_SAVEFAILED);
+			if (!memoryCard[slot]->Save()) PanicAlert(E_SAVEFAILED);
```

Comment:
E_SAVEFAILED is just "File write failed", which seems like something we want to convey to the user.